flask_app/controllers/tasks.py

```diff
@@ -12,14 +12,14 @@
 def post_task():
     return render_template("post.html")
 
-@app.route('/dashboard')
```

Comment:
@aacole9 this route was duplicated so I commented this one out in favor of the controller project dashboard route.